hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -486,7 +487,9 @@ public void onExecutionFailure(Throwable e) {
      */
     public void logError(Throwable e) {
         final ILogger logger = getLogger();
-        if (e instanceof RetryableException) {
+        if (e instanceof SilentException) {
+            logger.finest(e.getMessage(), e);
+        } else if (e instanceof RetryableException) {
```

Comment:
One could mark an `Exception` with both `SilentException` & `RetryableException`; in this case it will not be retried at all, just silently ignored. I cannot think why one would want a silent & retryable exception, but should we add a related warning in `SilentException`'s javadoc?
Also, since `QuorumException` becomes a `SilentException` with this PR, the `instanceof QuorumException` else-if branch below can be removed.